client/src/components/ForgotPassword/style.scss

```diff
@@ -0,0 +1,297 @@
+.login-container {
+  .container-fluid {
+    display: flex;
+    justify-content: flex-start;
+    max-width: 1440px;
+    padding: 0;
+    background: $white;
+  }
+
+  .bg-img {
+    position: relative;
+    width: 100%;
+    height: 100vh;
+    max-width: 640px;
+    overflow: hidden;
+
+    img {
+      width: 100%;
+      height: 100%;
+      max-width: 640px;
+
+      @media screen and (max-width: $mobile-width) {
+        display: none;
+      }
+    }
+
+    .logo-in-sign-up {
+      position: absolute;
+      width: 253px;
+      height: 83px;
+      text-align: center;
+      background: $crimson;
+
+      h1 {
+        margin: 0;
+        font-family: Open Sans, serif;
+        font-size: 1.5rem;
+        font-style: normal;
+        font-weight: 600;
+        line-height: 83px;
+        color: $rose-white;
+      }
+
+      @media screen and (max-width: $mobile-width) {
+        z-index: 2;
+        width: 100%;
+        max-width: $mobile-width;
+      }
+    }
+
+    @media screen and (max-width: $mobile-width) {
+      width: 100%;
+      max-width: $mobile-width;
+    }
+  }
+
+  .forgot-password-form {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 100%;
+    height: 100%;
+    max-width: 800px;
+
+    .get-started {
+      display: flex;
+      align-items: center;
+      justify-content: space-around;
+      width: 100%;
+      height: 83px;
+      max-width: 375px;
+      padding: 30px 0;
+      margin: 0 20px 0 auto;
```

Comment:
Interesting margin. How does this `auto` influence on space on the left?

Reply:
pushing block with button "Get started" to the right side